[2025-09-01 00:38:20,174] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather manual__2025-09-01T00:38:19.603014+00:00 [queued]>
[2025-09-01 00:38:20,203] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather manual__2025-09-01T00:38:19.603014+00:00 [queued]>
[2025-09-01 00:38:20,204] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-09-01 00:38:20,204] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-09-01 00:38:20,204] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-09-01 00:38:20,247] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): fetch_and_send_weather> on 2025-09-01 00:38:19.603014+00:00
[2025-09-01 00:38:20,259] {standard_task_runner.py:52} INFO - Started process 3278 to run task
[2025-09-01 00:38:20,265] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'fetch_and_send_weather', 'manual__2025-09-01T00:38:19.603014+00:00', '--job-id', '398', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmp55st78qw', '--error-file', '/tmp/tmpgkvh1648']
[2025-09-01 00:38:20,267] {standard_task_runner.py:80} INFO - Job 398: Subtask fetch_and_send_weather
[2025-09-01 00:38:20,385] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.fetch_and_send_weather manual__2025-09-01T00:38:19.603014+00:00 [running]> on host 5d7670b258c3
[2025-09-01 00:38:20,524] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=fetch_and_send_weather
AIRFLOW_CTX_EXECUTION_DATE=2025-09-01T00:38:19.603014+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-09-01T00:38:19.603014+00:00
[2025-09-01 00:38:20,525] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-09-01 00:38:20,526] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/***/scripts && python3 kafka/producer.py']
[2025-09-01 00:38:20,543] {subprocess.py:85} INFO - Output:
[2025-09-01 00:38:21,752] {subprocess.py:92} INFO - Envoi vers Kafka : {'temperature': 28, 'humidity': 84, 'windSpeed': 0, 'cloudCover': 100, 'rainIntensity': 0, 'timestamp': '2025-09-01T00:38:00Z'}
[2025-09-01 00:38:21,822] {subprocess.py:96} INFO - Command exited with return code 0
[2025-09-01 00:38:21,875] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=fetch_and_send_weather, execution_date=20250901T003819, start_date=20250901T003820, end_date=20250901T003821
[2025-09-01 00:38:21,923] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-09-01 00:38:21,982] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
